build-customization.rst

```diff
@@ -0,0 +1,72 @@
+.. post:: Nov 10, 2022
+   :tags: announcement, feature, builds
+   :author: Manuel
+   :location: BCN
+
+.. meta::
+   :description lang=en:
+      We released a new config key (``build.commands``) to specify user-defined commands
+      which allows users to overwrite the build process completely.
+
+
+Overwrite the build process completely
+======================================
+
+We are happy to announce a new *beta* feature that allow users to **overwrite the build process completely**.
+In the past, `we talked about executing custom commands <https://blog.readthedocs.com/user-defined-build-jobs/>`_ *in-between* the Read the Docs build process.
+However, that approach may not be sufficient for some projects with a heavily customized build process,
```

Comment:
Is it just heavily customized builds? Or just any tool other than Sphinx or Mkdocs? We should likely mention both use cases.

Reply:
I added a mention to Pelican, Docsify and Docusaurus in the introduction.